ow_lander/src/ow_lander/trajectory_planner.py

```diff
@@ -0,0 +1,999 @@
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
```

Comment:
A lot of refactoring seems possible within this file, but I imagine some of it must wait until more arm actions are brought into the new package structure.

Reply:
It is partly that, but mostly that there is not enough time to include a refactor of this at present and also make our deadline for command unification.